<?xml version="1.0"?>
<launch>
  <rosparam file="$(find eurobot_loc)/launch/secondary_robot_params.yaml" command="load" ns="secondary_robot"/>
  <param name = "field/color" value = "orange"/>
  <include file="$(find eurobot_sim)/launch/visualization.launch" />
  <group ns = "secondary_robot">
          <node pkg="eurobot" name="particle_filter_node" type="particle_filter_node.py" respawn="true" output="screen"/>
  </group>
</launch>

